taskcluster/fenix_taskgraph/worker_types.py

```diff
@@ -157,3 +157,56 @@ def build_github_release_payload(config, task, task_def):
         "{}:github:project:{}".format(scope_prefix, worker["github-project"]),
         "{}:github:action:{}".format(scope_prefix, worker["action"]),
     ])
+
+
+@payload_builder(
+    "scriptworker-tree",
+    schema={
+        # FIXME: do we need upstream artifacts?
```

Comment:
I don't think we do.

Reply:
I was trying to get rid of upstream-artifacts and wasn’t sure how. But I realized in the meantime, upstream-artifacts is being added in my task given https://github.com/mozilla-mobile/fenix/blob/master/taskcluster/fenix_taskgraph/transforms/multi_dep.py#L74 so I can simply switch in the payload from Required to Optional for now.